src/main/java/uk/gov/hmcts/dts/fact/services/admin/AdminCourtAreasOfLawService.java

```diff
@@ -20,38 +20,47 @@
 
     private final CourtRepository courtRepository;
     private final CourtAreaOfLawRepository courtAreaOfLawRepository;
+    private final CourtAreaOfLawSpoeRepository courtAreaOfLawSpoeRepository;
 
     @Autowired
-    public AdminCourtAreasOfLawService(final CourtRepository courtRepository, final CourtAreaOfLawRepository courtAreaOfLawRepository) {
+    public AdminCourtAreasOfLawService(final CourtRepository courtRepository, final CourtAreaOfLawRepository courtAreaOfLawRepository,
+                                       final CourtAreaOfLawSpoeRepository courtAreaOfLawSpoeRepository) {
         this.courtRepository = courtRepository;
         this.courtAreaOfLawRepository = courtAreaOfLawRepository;
+        this.courtAreaOfLawSpoeRepository = courtAreaOfLawSpoeRepository;
     }
 
     public List<AreaOfLaw> getCourtAreasOfLawBySlug(final String slug) {
 
         Court court = courtRepository.findBySlug(slug).orElseThrow(() -> new NotFoundException(slug));
         return courtAreaOfLawRepository.getCourtAreaOfLawByCourtId(court.getId())
             .stream()
-            .map(aol -> new AreaOfLaw(aol.getAreaOfLaw(), aol.getSinglePointOfEntry()))
+            .map(aol -> new AreaOfLaw(aol.getAreaOfLaw(),
+                                      courtAreaOfLawSpoeRepository.getAllByCourtIdAndAreaOfLawId(court.getId(), aol.getAreaOfLaw().getId())
```

Comment:
The admin portal spoe tab is now separate from the cases heard tab, which means the cases heard tab now do not need to know about spoe? We are still returning the spoe for the Admin area of law endpoints. Is that because you want this same model class to be used in the future when we add the admin spoe tab?

Reply:
Yeah pretty much! I imagine when we have the tab for altering the spoe for an area of law, it'd be best to reuse one area of law model, rather than two?